tslint.json

```diff
@@ -0,0 +1,154 @@
+{
+  "rulesDirectory": [
+    "node_modules/tslint-eslint-rules/dist/rules"
+  ],
+  "extends": "tslint:latest",
+  "rules": {
+    "class-name": true,
+    "comment-format": [
+      true,
+      "check-space"
+    ],
+    "eofline": true,
+    "indent": [
+      true,
+      "tab"
```

Comment:
Why not `space` ?

Reply:
Tabs option allows adding of spaces without ignoring ts rules, usually 2 and 4 spaces before every line of code